ui/tests/acceptance/clients/counts/overview-test.js

```diff
@@ -288,12 +278,11 @@ module('Acceptance | clients | overview | sync in license, not activated', funct
     assert.dom(GENERAL.tab('sync')).exists('sync tab is shown because feature is in license');
   });
 
-  test('it should hide secrets sync charts', async function (assert) {
+  test('it should hide secrets sync stats', async function (assert) {
     assert
-      .dom(CHARTS.chart('Secrets sync usage'))
-      .doesNotExist('chart is hidden because feature is not activated');
-
-    assert.dom('[data-test-stat-text="secret-syncs"]').doesNotExist();
+      .dom(CLIENT_COUNT.statTextValue('Secret sync'))
+      .doesNotExist('stat is hidden because feature is not activated');
+    assert.dom(CLIENT_COUNT.statTextValue('Entity')).exists('other stats are still visible');
```

Comment:
we were previously making a `doesNotExist` assertion on the wrong element. oops! i fixed that throughout this file and added a related assertion to ensure we don't get a falsely passing tests (i.e. if _all_ the chart stats were missing for some reason).

Reply:
Great catch!